frontend/src/services/applicationService.js

```diff
@@ -183,4 +183,41 @@ export default {
     )
     return results.some((result) => result)
   },
+
+  async createEmployeeCertificate(payload) {
+    try {
+      if (isEmpty(payload)) return
+      const response = await ApiService.apiAxios.post(`${ApiRoutes.APPLICATIONS}/employeeCertificate`, payload)
```

Comment:
Can we create service methods for these instead of calling the generic ApiService?

Reply:
sorry @weskubo-cgi, probably my brain is not working on Monday. Could you please clarify this comment? The createEmployeeCertificate() is a service method in applicationService.

Reply:
No, I'm losing it. Didn't notice what file I was looking at!